test/raw_explain/test_data_mapper.py

```diff
@@ -53,6 +53,14 @@ def test_column_with_brackets(self):
         result = data_mapper.transform(x)
         assert result.shape == (2, 2)
 
+    def test_large_sparse_transformation(self):
+        x = np.linspace((1,) * 8000, (10,) * 8000, 10)
+        encoder = OneHotEncoder()
+        encoder.fit(x)
+        data_mapper = DataMapper([(np.arange(8000), encoder)])
+        result = data_mapper.transform(x)
+        assert result.shape == (10, 80000)
```

Comment:
Might be good to do a values check as well to make sure the array contents are as expected for some input

Reply:
done!